sdcm/utils/issues.py

```diff
@@ -81,6 +105,18 @@ def get_issue_details(self, issue):
                                severity=Severity.WARNING,
                                trace=sys._getframe().f_back).publish()  # pylint: disable=protected-access
             return None
+        try:
+            if issue_details := self.cache.get_issue(owner=issue_parsed.user_id, repo_id=issue_parsed.repo_id, issue_id=issue_parsed.issue_id):
+                labels = [dict(name=label) for label in issue_details['labels'].strip().rstrip('|').split('|')]
```

Comment:
this can move to CachedGitHubIssues (that would be copied into dtest)

Reply:
+1

Reply:
moved it